dbt/adapters/bigquery/impl.py

```diff
@@ -662,7 +662,7 @@ def load_dataframe(self, database, schema, table_name, agate_table, column_overr
         load_config = google.cloud.bigquery.LoadJobConfig()
         load_config.skip_leading_rows = 1
         load_config.schema = bq_schema
-
+        load_config.field_delimiter=field_delimiter
```

Comment:
The [pre-commit hook CI check](https://github.com/dbt-labs/dbt-bigquery/actions/runs/8056514676/job/22006359111?pr=1122) wants this format instead:
```suggestion
        load_config.field_delimiter = field_delimiter
```